### ISSUE TITLE
False positive on indentation with try/catch and typealias

### ISSUE THREAD

**Issue by eygraber** [detekt/detekt#5720]
With the latest snapshot:

```
      val failure = try {
        client.newCall(request).execute().closeSilently()
        null
      }
      catch(t: Throwable) {
        t
      }
```

gets corrected to:

```
      val failure = try {
        client.newCall(request).execute().closeSilently()
        null
      }
        catch(t: Throwable) {
          t
        }
```

and

```
typealias MutableSurveyInterviewResponse =
  HashMap<SurveyInterviewResponseId, SurveyInterviewResponseValue?>
```

gets corrected to:

```
typealias MutableSurveyInterviewResponse =
HashMap<SurveyInterviewResponseId, SurveyInterviewResponseValue?>
```

ktlint 0.48.1 doesn't have this false positive.

## Your Environment

* Version of detekt used: Latest snapshot
* Version of Gradle used (if applicable): 8.0-rc-1
* Operating System and version: Ubuntu 22.04

**Comment by eygraber** [comment]
I tested this with the wrong version of ktlint (I had an old one on my path). Filed in ktlint as https://github.com/pinterest/ktlint/issues/1788
